tasks/configure.yml

```diff
@@ -7,8 +7,17 @@
     path: "{{ etc_prefix }}/haproxy/frontends.d"
     state: directory
 
-- name: 'Empty the folder if not already empty'
-  command: find {{ etc_prefix }}/haproxy/frontends.d -name *.cfg -exec rm -f {} \;
+- name: "List files for the frontends"
+  shell: ls -1 "{{ etc_prefix }}/haproxy/frontends.d"
```

Comment:
Use the [`find` module](https://docs.ansible.com/ansible/find_module.html) instead of [parsing `ls`](http://mywiki.wooledge.org/ParsingLs).